docs/src/01-data-sources.jmd

```diff
@@ -128,11 +139,17 @@ and run through the bioBakery metagenome workflow.
 Each sample was sequenced on 4 lanes of Illumina {{TODO: add info about Illumina}}
 generating paired-end reads.
 
+**Feel free to steal from what I've written in the Methods section thus far**
```

Comment:
Instead, I think we should just link to the relevant section of the manuscript. No reason to duplicate effort (and otherwise we'd have to track if things change in one place but not the other etc)